<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>464</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string notr="true">easyp2p</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="group_box_platform_top">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="title">
       <string>For which P2P platforms should the results be loaded?</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="group_box_platforms">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_2">
              <property name="spacing">
               <number>6</number>
              </property>
              <item row="0" column="0">
               <widget class="QCheckBox" name="check_box_bondora">
                <property name="text">
                 <string notr="true">Bondora</string>
                </property>
                <property name="shortcut">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QCheckBox" name="check_box_mintos">
                <property name="text">
                 <string notr="true">Mintos</string>
                </property>
                <property name="shortcut">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QCheckBox" name="check_box_dofinance">
                <property name="text">
                 <string notr="true">DoFinance</string>
                </property>
                <property name="shortcut">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QCheckBox" name="check_box_peerberry">
                <property name="text">
                 <string notr="true">PeerBerry</string>
                </property>
                <property name="shortcut">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QCheckBox" name="check_box_estateguru">
                <property name="text">
                 <string notr="true">Estateguru</string>
                </property>
                <property name="shortcut">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QCheckBox" name="check_box_robocash">
                <property name="text">
                 <string notr="true">Robocash</string>
                </property>
                <property name="shortcut">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QCheckBox" name="check_box_grupeer">
                <property name="text">
                 <string notr="true">Grupeer</string>
                </property>
                <property name="shortcut">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QCheckBox" name="check_box_swaper">
                <property name="text">
                 <string notr="true">Swaper</string>
                </property>
                <property name="shortcut">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QCheckBox" name="check_box_iuvo">
                <property name="text">
                 <string notr="true">Iuvo</string>
                </property>
                <property name="shortcut">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QCheckBox" name="check_box_twino">
                <property name="text">
                 <string notr="true">Twino</string>
                </property>
                <property name="shortcut">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="check_box_select_all">
           <property name="text">
            <string>Select/deselect all</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_date_range">
      <item>
       <widget class="QGroupBox" name="groupBox_start_date">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="title">
         <string>Start date</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QComboBox" name="combo_box_start_month"/>
           </item>
           <item>
            <widget class="QComboBox" name="combo_box_start_year"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_end_date">
        <property name="title">
         <string>End date</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QComboBox" name="combo_box_end_month"/>
           </item>
           <item>
            <widget class="QComboBox" name="combo_box_end_year"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="title">
       <string>Where should the results be saved?</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLineEdit" name="line_edit_output_file">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="push_button_file_chooser">
           <property name="text">
            <string>Choose File</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="push_button_start">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="text">
         <string>Start evaluation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tool_button_settings">
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>736</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>&amp;Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionGerman"/>
   </widget>
   <addaction name="menuLanguage"/>
  </widget>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionGerman">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>German</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
